src/Microsoft.Diagnostics.EventFlow.Core/Implementations/EventData.cs

```diff
@@ -28,6 +29,8 @@ public class EventData: IDeepCloneable<EventData>
 
         public long Keywords { get; set; }
 
+        public object EventContext { get; set; } = ContextResolver.GetRequestContext<object>();
```

Comment:
The problem is that not all inputs support any kind of context. 
1. Trace: no context
2. EventSource: has payload, but it's mapped to EventData.Payload
3. PerformanceCounter: context does not make sense
4. Serilog proprieties are implemented as payload
5. ILogger supports state and consistent approach would be to pass it in the payload

It makes me think that 
1. for most inputs (which support some kind of context), payload is existing way to pass it
2. `TraceInput` may get the context with `ContextResolver.GetContext<object>()` and put it in the payload, But I really doubt that we should do it for the inputs which do not have any context by it's nature.